From: SeongJae Park <sj@kernel.org>
Subject: mm/damon/vaddr: remove an unnecessary warning message

The DAMON virtual address space monitoring primitive prints a warning
message for wrong DAMOS action.  However, it is not essential as the code
returns appropriate failure in the case.  This commit removes the message
to make the log clean.

Link: https://lkml.kernel.org/r/20211201150440.1088-5-sj@kernel.org
Fixes: 6dea8add4d28 ("mm/damon/vaddr: support DAMON-based Operation Schemes")
Signed-off-by: SeongJae Park <sj@kernel.org>
Cc: Brendan Higgins <brendanhiggins@google.com>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/damon/vaddr.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/damon/vaddr.c~mm-damon-vaddr-remove-an-unnecessary-warning-message
+++ a/mm/damon/vaddr.c
@@ -626,7 +626,6 @@ int damon_va_apply_scheme(struct damon_c
 	case DAMOS_STAT:
 		return 0;
 	default:
-		pr_warn("Wrong action %d\n", scheme->action);
 		return -EINVAL;
 	}
 
_
